autoload/OmniSharp.vim

```diff
@@ -142,20 +147,25 @@ function! OmniSharp#NavigateUp() abort
   call OmniSharp#CheckPyError()
 endfunction
 
-" Find the solution for this file.
-" Caches result
-function! OmniSharp#FindSolution(...) abort
+" Find the solution or directory for this file.
+function! OmniSharp#FindSolutionOrDir(...) abort
   let interactive = a:0 ? a:1 : 1
   let bufnum = a:0 > 1 ? a:2 : bufnr('%')
-  if empty(getbufvar(bufnum, 'OmniSharp_sln_file'))
-    try
-      let sln = s:FindSolution(interactive, bufnum)
-    catch e
-      return ''
-    endtry
-    call setbufvar(bufnum, 'OmniSharp_sln_file', sln)
+  if empty(getbufvar(bufnum, 'Omnisharp_buf_server'))
```

Comment:
Please rename this variable to `OmniSharp_buf_server` (capital "S"), so it is consistent with all other variables and usages of "OmniSharp"